Uno.Gallery/Uno.Gallery.Shared/Views/Styles/NeumorphicStyles.xaml

```diff
@@ -1,49 +1,541 @@
 ﻿<ResourceDictionary xmlns="http://schemas.microsoft.com/winfx/2006/xaml/presentation"
 					xmlns:x="http://schemas.microsoft.com/winfx/2006/xaml"
 					xmlns:utu="using:Uno.Toolkit.UI"
+					xmlns:converters="using:Uno.Gallery.Converters"
+					xmlns:not_mobile="http://schemas.microsoft.com/winfx/2006/xaml/presentation"
+					xmlns:win="http://schemas.microsoft.com/winfx/2006/xaml/presentation"
 					xmlns:um="using:Uno.Material">
 
+	<!--#region Shadow ThemeDictionaries-->
 	<ResourceDictionary.ThemeDictionaries>
 		<ResourceDictionary x:Key="Dark">
-
+			<!-- Workaround for OutlineVariantBrush not found in the NeumorphicColorPaletteOverride -->
+			<SolidColorBrush x:Key="OutlineVariantBrush"
+							 Color="{ThemeResource OutlineVariantColor}"/>
```

Comment:
This is defined in the NeumorphicColorsOverride.xaml as well, is that intended?

Reply:
We should be using the one from NeumorphicColorsOverride.xaml, but since `OutlineVariantBrush` isn't defined on Themes we can't override it so using the one from NeumorphicColorsOverride.xaml won't work. 

I left it there for reference, but if you think it's better to remove for now we can remove it.

Reply:
I think you can remove it in NeumorphicColorsOverride.xaml until it is fixed in Uno.Themes @eriklimakc, after it will be fixed, you will be able to remove it from NeumorphicStyles.xaml and add it to NeumorphicColorsOverride.xaml

Reply:
Fixed in @eriklimakc last commit